frontend/src/app/components/auth-page/auth-page.component.sass

```diff
@@ -0,0 +1,132 @@
+@import 'src/colors.sass'
+
+.auth-container
+  height: 100vh
+  min-height: 100%
+  width: 100%
+  min-width: 100%
+  display: grid
+  grid-template-columns: 3fr 2fr
+  align-items: center
+  margin: -10px -10px -10px -10px
+
+.logo-container
+  height: 100%
+  background-position: center
+  background-repeat: no-repeat
+  background-size: cover
+  background-image: url('/assets/background.png')
+  cursor: pointer
```

Comment:
Remove "cursor: pointer" rule, it is not useful now

Reply:
done